demo/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="../dist/responsive-grid.css">
+  <link rel="stylesheet" href="./demo-styles.css">
+</head>
+
+<body style="background-color: #e5e5e5;">
+  <div class="container">
+    <div class="row">
+      <div class="col xl-10 lg-6 md-8">8</div>
+      <div class="col xl-2 lg-6 md-4">4</div>
+      <div class="col xl-2 md-4">4</div>
+      <div class="col xl-10 md-8">8</div>
+    </div>
+  </div>
+</body>
+
```

Comment:
Do you think we should add more examples? Or we create it after?

Reply:
Won't we (for now) use this file for tests? if that's the case we can create more examples later, but if it's not, it would be better to create more

Reply:
Let's do it, but not in this PR.